cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlannerTest.java

```diff
@@ -204,7 +230,7 @@ public void testClear() {
                                           Collections.<String>emptySet());
 
     planner.addExecutionProposals(proposals, expectedCluster);
-    assertEquals(1, planner.remainingDataToMoveInMB());
+    assertEquals(4, planner.remainingDataToMoveInMB());
```

Comment:
No, it is because I change to the partition size of `partitionMovement1`'s partition(`TOPIC2`) from 1 to 4, and the reason for this change is to better test `PrioritizeLargeReplicaMovementStrategy`.

Reply:
No, it is because I change to the partition size of partitionMovement1's partition(TOPIC2) from 1 to 4, and the reason for this change is to better test `PrioritizeLargeReplicaMovementStrategy`.